apps/configurator/priv/repo/seeds.exs

```diff
@@ -0,0 +1,5 @@
+## TODO: Change this once PR#336 is merged
```

Comment:
What's this comment for? #336 is closed 🤔

Reply:
Removed seeds (content), it will be dynamically done at runtime cause it makes sense and its easier